server/app.js

```diff
@@ -33,6 +33,8 @@ if (process.env.NODE_ENV === 'development' || !process.env.NODE_ENV) {
     app.use(cors(corsOptions));
 }
 
+console.log('NODE_ENV: ', process.env.NODE_ENV);
+
```

Comment:
is this intentional?

Reply:
yes, i'll take it out after this is deployed to digital ocean/heroku, I want to see what the env variable is set to in those instances (I'm guessing 'production' but want to double check)